umm-spec-lib/src/cmr/umm_spec/test/expected_conversion.clj

```diff
@@ -8,6 +8,7 @@
     [clojure.string :as str]
```

Comment:
Hey, while you're in here, could you un-indent these requires by one space (to match our convention).